radicle-httpd/src/raw.rs

```diff
@@ -0,0 +1,54 @@
+use std::sync::Arc;
+
+use axum::extract::State;
+use axum::response::IntoResponse;
+use axum::routing::get;
+use axum::Router;
+
+use radicle::prelude::Id;
+use radicle::profile::Profile;
+use radicle::storage::git::paths;
+use radicle_surf::{Oid, Repository};
+
+use crate::api::axum_extra::Path;
+use crate::api::error::Error;
```

Comment:
we have resorted to using some stuff from `::api::`, any thoughts on this? (since we're no longer in `/api` but a level higher, in `/raw`)

Reply:
Maybe better to move these out to the top level then?

Reply:
so there's:
1. `crate::api::axum_extra::Path`
2. `crate::api::error::Error`

first one we can safely move out. second one, only if we want all errors to be jsons, is that something we'd want for `/git` and `/raw`?

Reply:
Oh I see hmm. No, for raw and git, I think we just want error codes for now, no error bodies.